src/store/gc.rs

```diff
@@ -0,0 +1,472 @@
+use std::{collections::HashSet, pin::Pin, sync::Arc};
```

Comment:
This has nothing to do with the PR, right?

Copies gc from fs to store top level. Probably related to something else?